Classes/Controller/BackendController.php

```diff
@@ -128,9 +135,14 @@ public function indexAction(NodeInterface $node = null)
 
         $siteNode = $node->getContext()->getCurrentSiteNode();
 
+        $clipboardSubject = $this->clipboardStateContainer->getClipboardSubject();
+        $clipboardMode = $this->clipboardStateContainer->getClipboardMode();
+
         $this->view->assign('user', $user);
         $this->view->assign('documentNode', $node);
         $this->view->assign('site', $siteNode);
+        $this->view->assign('clipboardSubject', $clipboardSubject);
+        $this->view->assign('clipboardMode', $clipboardMode);
```

Comment:
If we store the clipboard state on the server, that state needs to be flushed when the CB is pasted. So instead of pushing the clipboard content to the client for every request, I would suggest to do that with an explicit request (that also flushes the CB state)

Reply:
Why? I think it makes sense to keep the last copied node in clipboard, so you could paste it multiple times. This is the way clipboards work in general, e.g. in operating systems.